smrf/tests/smrf_test_case_lakes.py

```diff
@@ -45,3 +45,71 @@ def tearDownClass(cls):
         folder = os.path.join(cls.base_config.cfg['output']['out_location'])
         if os.path.exists(folder):
             shutil.rmtree(folder)
+
+    def assert_list(self, list1, list2):
+        """Couldn't get the assert_list to work"""
+
+        self.assertTrue(sorted(list1) == sorted(list2))
+
+    def assert_thread_variables(self):
```

Comment:
Would this be worth splitting into a test case for each distribution?

Reply:
The RME case will have two separate cases of thread variables, one for stations and gridded. Since the Lakes only uses HRRR, it's a little easier problem. I don't know if its worth it to fully flush this out.